WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1647,11 +1650,35 @@ public void onImageSettingsRequested(EditorImageMetaData editorImageMetaData) {
     @Override public void onImagePreviewRequested(String mediaUrl) {
 //        MediaPreviewActivity.showPreview(this, null, mediaUrl);
         // TODO: Temporarily open image editor at this point
-        ActivityLauncher.openImageEditor(this, mediaUrl);
+        String resizedImageUrl = StringUtils.notNullStr(mediaUrl);
```

Comment:
I considered moving this logic to the `PhotonUtil` class inside `WordPressUtils` lib.  Wondering if we can also move `ReaderUtils`  to the lib and reuse `ReaderUtils.getResizedImageUrl`?  I guess it will be out of scope of this PR.

Reply:
I'd consider moving the method we need from ReaderUtils to a utility class located in WordPressUtils lib. Wdyt?

SideNote: I'm also not sure why we need to introduce something like this here - we are loading images in the reader, Aztec, Gutenberg, comments. How come we don't have this logic somewhere already?

Reply:
Changed here d151f4f